Packs/Sixgill-Darkfeed/Integrations/Sixgill_DVE_Feed/Sixgill_DVE_Feed.py

```diff
@@ -0,0 +1,210 @@
+import traceback
+import requests
+
+from sixgill.sixgill_request_classes.sixgill_auth_request import SixgillAuthRequest
+from sixgill.sixgill_feed_client import SixgillFeedClient
+from sixgill.sixgill_constants import FeedStream
+from sixgill.sixgill_utils import is_indicator
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+""" CONSTANTS """
+CHANNEL_CODE = "7698e8287dfde53dcd13082be750a85a"
+MAX_INDICATORS = 1000
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+SUSPICIOUS_FEED_IDS = ["darkfeed_003"]
+DEMISTO_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+VERIFY = not demisto.params().get("insecure", True)
+SESSION = requests.Session()
+DESCRIPTION_FIELD_ORDER = OrderedDict(
+    [
+        ("Description", "eventdescription"),
+        ("Created", "creationdate"),
+        ("Modified", "modified"),
+        ("External id", "externalid"),
+        ("Sixgill DVE score - current", "sixgilldvescorecurrent"),
+        ("Sixgill DVE score - highest ever date", "sixgilldvescorehighesteverdate"),
+        ("Sixgill DVE score - highest ever", "sixgilldvescorehighestever"),
+        ("Sixgill - Previously exploited probability", "sixgillpreviouslyexploitedprobability"),
+        ("Event Name", "eventname"),
+        ("Event Type", "eventtype"),
+        ("Event Action", "eventaction"),
+        ("Previous level", "previouslevel"),
+        ("Event Description", "eventdescription"),
+        ("Event Datetime", "eventdatetime"),
+        ("CVSS 3.1 score", "cvss31score"),
+        ("CVSS 3.1 severity", "cvss31severity"),
+        ("NVD Link", "nvdlink"),
+        ("NVD - last modified date", "nvdlastmodifieddate"),
+        ("NVD - publication date", "nvdpublicationdate"),
+        ("CVSS 2.0 score", "cvss20score"),
+        ("CVSS 2.0 severity", "cvss20severity"),
+        ("NVD Vector - V2.0", "nvdvectorv20"),
+        ("NVD Vector - V3.1", "nvdvectorv31"),
+    ]
+)
+
+
+""" HELPER FUNCTIONS """
+
+
+def module_command_test(*args):
+    """
+    Performs basic Auth request
+    """
+    response = SESSION.send(
+        request=SixgillAuthRequest(
+            demisto.params()["client_id"], demisto.params()["client_secret"], CHANNEL_CODE
+        ).prepare(),
+        verify=VERIFY,
+    )
+    if not response.ok:
+        raise Exception("Auth request failed - please verify client_id, and client_secret.")
+    return "ok", None, "ok"
+
+
+def get_description(fileds_obj):
+    description_string = ""
+    for name, sixgill_name in DESCRIPTION_FIELD_ORDER.items():
+        description_string += f"{name}: {fileds_obj.get(sixgill_name)}\n"
+    fileds_obj["description"] = description_string
+    return fileds_obj
+
+
+def create_fields(stix_obj, event_obj, nvd_obj, score_obj, ext_id):
+    fields = {}
+    try:
+        fields = {
+            "description": "",
+            "creationdate": stix_obj.get("created", ""),
+            "modified": stix_obj.get("modified", ""),
+            "externalid": ext_id,
```

Comment:
Looking at the code, it looks like there is a scenario that ext_id is an empty string. just making sure you are aware and tested this case.